about.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="Preston Wealth Advisors" />
+    <title>Preston Wealth Advisors</title>
+    <link rel="stylesheet" href="reset.css" type="text/css" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,700,800' rel='stylesheet' type='text/css'>
+  </head>
+
+  <body>
+
+    <header class="header-home">
+      <div class="margin">
```

Comment:
This class name is too vague. What type of margin? Top, Bottom, Left, Right? What is the px count, or is it percentage?